src/frontend/src/binder/expr/function.rs

```diff
@@ -331,6 +331,7 @@ impl Binder {
                 ("ceil", raw_call(ExprType::Ceil)),
                 ("floor", raw_call(ExprType::Floor)),
                 ("abs", raw_call(ExprType::Abs)),
+                ("exp", raw_call(ExprType::Exp)),
```

Comment:
PG allows spelling both `pow` and `power`. We can add one line here to dispatch `power` to `ExprType::Pow`.